python/run-tests.py

```diff
@@ -162,7 +162,7 @@ def run_individual_python_test(target_dir, test_name, pyspark_python):
 
 
 def get_default_python_executables():
-    python_execs = [x for x in ["python2.7", "python3.4", "pypy"] if which(x)]
+    python_execs = [x for x in ["python2.7", "python3.4", "python3.5", "pypy"] if which(x)]
```

Comment:
i will push another commit at some point in the future to remove the reference to python3.4.

Reply:
Just wondering why not replace python3.4 now? From the steps in jira, there will be a symlink for 3.4 -> 3.5 so it would be running the same tests twice right?